components/date-picker/demo/locale.md

```diff
@@ -1,10 +1,19 @@
 ---
 order: 10
-title: 国际化
+title: 
+  zh-CN: 国际化
+  en-US: Locale
 ---
 
+## zh-CN
+
 通过 `locale` 配置时区、语言等, 默认支持 `en_US`，`zh_CN`。
 
+## en-US
+
+To set the properties like time zone, language and etc. en_US, zh_CN are supported by default.
```

Comment:
Use `locale` to set the properties like time zone, language and etc. `en_US`, `zh_CN` are supported by default.